docs/tutorials/generic_field_type/7_add_a_validation.md

```diff
@@ -1,35 +1,9 @@
 # Step 7 - Add basic validation
 
-!!! tip
-
-    You can find all files used and modified in this step on [GitHub](https://github.com/ezsystems/generic-field-type-tutorial/tree/Step_7).
-
 To provide basic validation that ensures both coordinates are provided, add assertions to the `src/FieldType/Point2D/Value.php`:
 
 ```php
-<?php
-
-use Symfony\Component\Validator\Constraints as Assert;
-
-final class Value implements ValueInterface
-
-{  
-   /**
-     * @var float|null
-     *
-     * @Assert\NotBlank()
-     */
-    private $x;
-     
-    /**
-     * @var float|null
-     *
-     * @Assert\NotBlank()
-     */
-    private $y;
-    
- // ...   
-}
+[[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Value.php', 6, 23) =]]
```

Comment:
I can see a slight problem here - the object has no ending parenthesis - I know where it comes from, we're pulling part of the file.

Would the following include work?

```
[[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Value.php', 6, 23) =]]
[[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Value.php', 57) =]]
```

 (Also, the first line is missing, is it on purpose? If not, would the following include work?)
 [[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Value.php', 0, 1) =]]